about.html

```diff
@@ -0,0 +1,104 @@
+            <div class="hamburger-container">
+                <div class="close"><img width="30px" id="closeBtn"
+                        src="images/67-679412_x-cross-close-symbol-icon-button-gui-grey.png" alt=""></div>
+
+                <ul class="hamburger-items ">
+                    <li><a href="index.html" class="items" id="menuabout">Home</a></li>
+                    <li><a href="" class="items" id="program">Program</a> </li>
+                    <li><a href="" class="items" id="join">Join</a></li>
+                    <li><a href="" class="items" id="Sponsor"> Sponsor</a></li>
+                    <li><a href="" class="items" id="enlist">Enlist</a> </li>
+                </ul>
+
+            </div>
+            <ul class="nav-items">
+                <li><a href="about.html" class="item" id="menuabout">About</a></li>
+                <li><a href="" class="item" id="program">Program</a> </li>
+                <li><a href="" class="item" id="join">Join</a></li>
+                <li><a href="" class="item" id="Sponsor"> Sponsor</a></li>
+                <li><a href="" class="item" id="enlist">Enlist</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <section class="about" id="about">
+        <div class="about-content">
+            <h1 class="about-header">The Global Musical Concert 2022</h1>
+            <div class="about-container">
+                <p class="about-text"> The Global music concert brings together the community of experts and high
+                    skilled musician.
+                    To show forth their musicianship.
+                    This year we hope to expand our invitation including upcoming musicians looking for a breakthrough
+                    So if you're active and engaged in the music world. We hope you'll join us this year to build a
+                    stronger, vibrant commons together
+                </p>
+            </div>
+            <div class="about-text2">
+                Please contact us per Email for any further questions about TGMC 2022!
+            </div>
+            <div> <a href="johnsonnifemi8@gmail.com" class="email">johnsonnifemi8@gmail.com</a> </div>
+        </div>
+    </section>
+    <div class="past-container">
+        <h1 class="past"> See past Global Musical Festival</h1>
+        <p class="look"> Take a look at the last two Global Music Festival which took place in New York and LA</p>
+        <div class="image-container">
+            <div class="one">
+                <div class="onn">
+                    <h1 class="one-date">2021</h1>
+                    <p class=" one-p">Global Music Festival 2021 in New York</p>
+                </div>
+            </div>
+            <div class="two">
+                <div class="twoo">
+                    <h1 class="two-date">2020</h1>
+                    <p class=" two-p">Global Music Festival 2020 in la</p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <footer>
```

Comment:
- [ ] you forgot to add the section that has the  logo of the website like in the design  so please try to add it in order to match the design 
![image](https://user-images.githubusercontent.com/61790587/175149480-b7ca8239-205f-458d-8a28-069e7656e2d0.png)